### PR TITLE
[SP-5069] Adding Backport of logic for PDI-17863 and PDI-18039

### PR DESCRIPTION
**Pull request by jpruiz114**
@pentaho-lmartins 

@gryphendowre @wseyler

### CODE REVIEW

File: engine/src/main/java/org/pentaho/di/www/GetStatusServlet.java [hunk 1]
```diff
@@ -294,7 +294,7 @@ public void doGet( HttpServletRequest request, HttpServletResponse response ) th
         out.println( "<table cellspacing=\"0\" cellpadding=\"0\" class=\"toolbar\" style=\"width: 100%; height: 26px; margin-bottom: 2px; border: 0;\">" );
         out.println( "<tbody><tr>" );
         out.println( "<td align=\"left\" style=\"vertical-align: middle; width: 100%\" id=\"trans-align\"></td>" );
-        out.println( "<td " + setupIconEnterLeaveJavascript( "pause" ) + " align=\"left\" style=\"vertical-align: middle;\"><div style=\"padding: 2px;\" onClick=\"resumeFunction( this )\" class=\"toolbar-button toolbar-button-disabled\" id=\"pause\"><img style=\"width: 22px; height: 22px\" src=\"" + prefix + "/images/run_option.svg\" title=\"" + run + "\"/></div></td>" );
+        out.println( "<td " + setupIconEnterLeaveJavascript( "pause" ) + " align=\"left\" style=\"vertical-align: middle;\"><div style=\"padding: 2px;\" onClick=\"resumeFunction( this )\" class=\"toolbar-button toolbar-button-disabled\" id=\"pause\"><img style=\"width: 22px; height: 22px\" src=\"" + prefix + "/images/run.svg\" title=\"" + run + "\"/></div></td>" );
```
Comment (pentaho-lmartins) [inline engine/src/main/java/org/pentaho/di/www/GetStatusServlet.java]: I can't find this change in the original PR which I guess is #6565.
Please confirm this is OK.

**Comment by jpruiz114** [comment]
@LeonardoCoelho71950 , @smmribeiro

File: engine/src/main/java/org/pentaho/di/www/GetStatusServlet.java (hunk 1 above)
Comment (jpruiz114) [reply engine/src/main/java/org/pentaho/di/www/GetStatusServlet.java]: That change is not included in the original code changes from PDI-17863 done by Dean DiRoma, but if you notice, the SVG filename was wrong under 8.2, causing a 404 for that image, and would differ from 8.3. Please advice @pentaho-lmartins if I should remove this change.

Comment (jpruiz114) [reply engine/src/main/java/org/pentaho/di/www/GetStatusServlet.java]: image

image

Comment (smmribeiro) [reply engine/src/main/java/org/pentaho/di/www/GetStatusServlet.java]: @jpruiz114 , did you test the behaviour before using the newly built jar?
There was a PR [pentaho-kettle#6081](https://github.com/pentaho/pentaho-kettle/pull/6081) back in December 2018 for "[SP-4800] Backport for BACKLOG-26741 -- 8.2 Carte status page unable to acess Scaleable Vector Graphic (.svg) files" that may already fixed that.
Using 8.2.0.6-QAT-655, I have this:
image

So it seems safe to remove that extra code.

Comment (smmribeiro) [reply engine/src/main/java/org/pentaho/di/www/GetStatusServlet.java]: @pentaho-lmartins , with the information that @jpruiz114 gathered (and left on [this comment](https://jira.pentaho.com/browse/SP-5069?focusedCommentId=370273&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-370273)), my opinion is to approve and merge this PR.
Do you agree?

**Pull request merged**